spec/controllers/api/v1/artists_controller_spec.rb

```diff
@@ -28,8 +28,40 @@
           expect(returned_json[1]["name"]).to  eq second_artist.name
           expect(returned_json[1]["bio"]).to  eq second_artist.bio
           
-          
-
         end  
       end 
+
+      describe "POST#create" do
+      it "creates a new artist" do
+          post_json = {
+              artist: {
+                  name: "The Luke Band",
+                  bio: "Pretty Sick"
+              }
+          }
+
+          prev_count = Artist.count
+          post(:create, params: post_json, format: :json)
+          expect(Artist.count).to eq(prev_count + 1)
+      end
+      it "returns the json of the newly posted artist" do
+          post_json = {
+              artist: {
+                  name: "The Luke Band",
+                  bio: "Pretty Sick"
+              }
+          }
```

Comment:
since you are using this "post_json" twice, you should make a let() variable for it in your describe block